src/sentry/incidents/action_handlers.py

```diff
@@ -364,13 +367,20 @@ def generate_incident_trigger_email_context(
     elif CRASH_RATE_ALERT_AGGREGATE_ALIAS in aggregate:
         aggregate = aggregate.split(f"AS {CRASH_RATE_ALERT_AGGREGATE_ALIAS}")[0].strip()
 
-    threshold = trigger.alert_threshold if is_active else alert_rule.resolve_threshold
-    if threshold is None:
-        # Setting this to trigger threshold because in the case of a resolve if no resolve
-        # threshold is specified this will be None. Since we add a comparison sign to the
-        # string it makes sense to set this to the trigger alert threshold if no threshold is
-        # specified
-        threshold = trigger.alert_threshold
+    threshold: None | str | float = None
+    if alert_rule.detection_type == AlertRuleDetectionType.DYNAMIC:
```

Comment:
just for my own learning, is this used for each notification channel or do we need to recreate it?

Reply:
the message builders are different for each integration, I'm making a PR for each